ttsai/rotator_verification.ipynb

```diff
@@ -0,0 +1,3831 @@
+{
```

Comment:
<p>Please add a test for negative minimum speed. Add plot son jerk/ torque</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='90'/>

Reply:
This is not included in the Ander's Simulink test case. This script is to translate what he has in the Simulink test.